github/teams.go

```diff
@@ -276,11 +316,63 @@ func (s *TeamsService) ListChildTeams(ctx context.Context, teamID int64, opt *Li
 	return teams, resp, nil
 }
 
-// ListTeamRepos lists the repositories that the specified team has access to.
+// ListChildTeamsByParentName lists child teams for a parent team given parent name.
```

Comment:
For consistency, please rename this function to `ListChildTeamsByParentSlug`, and change the comment to say "given parent slug".